.buildkite/pipeline.yml

```diff
@@ -0,0 +1,13 @@
+steps:
+  - label: Build and Upload to TestFlight
+    command: .buildkite/commands/build-and-upload-to-testflight.sh
+    plugins:
+      - automattic/a8c-ci-toolkit#3.4.2
+      - automattic/nvm#0.3.0
+    agents:
+      queue: mac
+    env:
+      IMAGE_ID: xcode-15.4
```

Comment:
Nitpick / (Premature?) optimization: Have you considered using the shared vars pattern?

I thought of that mainly because I like how we can use it to move the Xcode version in `.xcode-version`, making it available to other tools. However, I don't know if there are any other tools that would be reading it in this repo.